mlrun/serving/server.py

```diff
@@ -681,6 +754,7 @@ def execute_graph(
 
     :param context: The job's execution client context.
     :param data: The input data to the job, to be pushed into the graph row by row, or in batches.
+    :param timestamp_column: The name of the column that will be used as the timestamp for model monitoring purposes.
```

Comment:
Lets add  this https://github.com/mlrun/mlrun/pull/8355#discussion_r2233830371, to the docs